terraform/aws/public-cloud/main.tf

```diff
@@ -35,6 +37,16 @@ module "aws-keypair" {
   public_key_filename = "${var.public_key_file}"
 }
 
+module "ca" {
```

Comment:
maybe rename this module to be a bit more explicit as its specifically traefik ca certs